pkg/resources/file_format.go

```diff
@@ -146,31 +148,37 @@ var fileFormatSchema = map[string]*schema.Schema{
 	"date_format": {
 		Type:        schema.TypeString,
 		Optional:    true,
+		Computed:    true,
```

Comment:
should this be computed? my understanding is that computed attributes should be read-only. Perhaps adding a default of "AUTO" would make more sense?

Reply:
Yes that was my assumption too but then I found [this stack overflow answer](https://stackoverflow.com/questions/57784666/how-should-a-terraform-provider-handle-default-values-applied-on-the-server-side). Turns out using optional + computed allows us to set a default value when it is not set by the user. 
- using optional alone means we can't set a default value without generating drift (tf expects a null value but Read returns something)
- using computed alone does not allow the user to set a value

Reply:
noted